src/index.ts

```diff
@@ -58,6 +58,8 @@ export class Textile {
   profile: Profile
   /** @property {Schemas} schemas - Manage Textile Schemas */
   schemas: Schemas
+  /** @property {Subscribe} subscribe - Subscribe to thread updates */
+  subscribe: Subscribe
```

Comment:
Calling this `subscribe` rather than `sub` because it feels more readible to an end user: `textile.subscribe.stream` reads like "subscribe to stream" to me... but I could be convinced to expose it as `sub` to stay consistent with cli...

Reply:
I like it. I can change the cli to match.

Reply:
Sounds good 👍